parcels/field.py

```diff
@@ -136,7 +136,8 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
         :param variable: Name of the field to create. Note that this has to be a string
         :param dimensions: Dictionary mapping variable names for the relevant dimensions in the NetCDF file
         :param indices: dictionary mapping indices for each dimension to read from file.
-               This can be used for reading in only a subregion of the NetCDF file
+               This can be used for reading in only a subregion of the NetCDF file.
+               Note that negative indices are not allowed.
```

Comment:
Are they really 'not allowed'? Or are they just discouraged as it makes the code much slower?